catroid/src/main/java/org/catrobat/catroid/content/bricks/LegoEv3MotorMoveBrick.java

```diff
@@ -151,8 +147,11 @@ public void onItemSelected(AdapterView<?> arg0, View arg1, int position, long ar
 			public void onNothingSelected(AdapterView<?> arg0) {
 			}
 		});
-
+		if (motorEnum == null) {
+			readResolve();
+		}
 		motorSpinner.setSelection(motorEnum.ordinal());
+		motorSpinner.setGravity(Gravity.CENTER);
```

Comment:
It does not have to be done in code but I wanted to add it so all LegoBricks look alike. This was the only brick that did not set the gravity of the spinner to center.
Option one: change all xmls of the bricks with spinners or
Option two: add one line in one file to make them all look alike.
I guess you would have liked option one. If you insist on it to be done that way let me know.